dal-identification.yml

```diff
@@ -49,22 +49,62 @@ paths:
             application/json:
               schema:
                 type: integer
-  /api/v1/autotransport/identification/customers?page=[page]&size=[size]:
+  /api/v1/autotransport/identification/customers?pageNo=[pageNo]&pageSize=[pageSize]&firstName=[firstName]&lastName=[lastName]&userName=[userName]&legalName=[legalName]&commonName=[commonName]&status=[status]&address=[address]&customBroker=[customBroker]:
```

Comment:
the same parameters should be in the customer/count as well except pageNo and pageSize.